collagraph/__init__.py

```diff
@@ -583,7 +583,12 @@ def equivalent_code(a, b):
     ]:
         attr_a = getattr(a, attr, None)
         attr_b = getattr(b, attr, None)
-        if attr_a != attr_b:
+        if attr in ["co_freevars", "co_varnames"]:
```

Comment:
just a stylistic thing, but in these situations I would recommend using sets, containment test is O(1)

Reply:
```suggestion
        if attr in {"co_freevars", "co_varnames"}:
```